conductor/lib/nuke_utils.py

```diff
@@ -107,6 +134,9 @@ def collect_dependencies(write_nodes, views, dependency_knobs=None):
     '''
     dependency_knobs = dependency_knobs or {}
     deps = set()
+    
+    # A custom OCIO config is defined on the root node
+    deps = deps.union(set(scrape_ocio_dependencies()))
```

Comment:
Since `deps` is an empty **set**, is there any need to `union`?

Reply:
good point